tools/deploy-to-gear/src/main.rs

```diff
@@ -0,0 +1,187 @@
+use clap::{Args, Parser, Subcommand};
+use gclient::{EventListener, EventProcessor, GearApi, WSAddress};
+use gear_core::ids::ProgramId;
+use sails_rs::{calls::Call, gclient::calls::GClientRemoting, prelude::*};
+use vft_client::{traits::Vft, Vft as VftClient};
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct Cli {
+    #[command(subcommand)]
+    command: CliCommands,
+}
+
+#[allow(clippy::enum_variant_names)]
+#[derive(Subcommand)]
+enum CliCommands {
+    /// Deploy extended vft contract
+    #[clap(visible_alias("ev"))]
+    ExtendedVft(ExtendedVftArgs),
+}
+
+#[derive(Args)]
+struct ExtendedVftArgs {
+    /// Address of the Gear RPC endpoint
+    #[arg(
+        long = "gear-endpoint",
+        default_value = "wss://testnet.vara.network",
+        env = "GEAR_RPC"
+    )]
+    gear_endpoint: String,
+    /// Port of the Gear RPC endpoint
+    #[arg(long = "gear-port", default_value = "443", env = "GEAR_PORT")]
+    gear_port: u16,
+
+    /// Name of the token that will be set during initialization
+    #[arg(long = "token-name", short = 'n', default_value = "VftToken")]
+    token_name: String,
+    /// Symbol of the token that will be set during initialization
+    #[arg(long = "token-symbol", short = 's', default_value = "VT")]
+    token_symbol: String,
+    /// Decimals of the token that will be set during initialization
+    #[arg(long = "token-decimals", short = 'd', default_value = "18")]
+    token_decimals: u8,
+
+    /// ActorId that will be allowed to mint new tokens
+    #[arg(long = "mint-admin")]
+    mint_admin: Option<String>,
+    /// ActorId that will be allowed to burn tokens
+    #[arg(long = "burn-admin")]
+    burn_admin: Option<String>,
+
+    /// Whether to NOT revoke the admin role after the deployment
+    #[arg(long, action)]
+    dont_revoke_admin_role: bool,
+}
+
+#[tokio::main]
+async fn main() {
+    let _ = dotenv::dotenv();
+
+    let cli = Cli::parse();
+
+    match cli.command {
+        CliCommands::ExtendedVft(args) => {
+            let address = WSAddress::new(&args.gear_endpoint, Some(args.gear_port));
+            let gear_api = GearApi::init(address)
+                .await
+                .expect("Failed to initialize GearApi");
+            let listener = gear_api
+                .subscribe()
+                .await
+                .expect("Failed to subscribe to listener");
+
+            let str_to_actorid = |s: String| {
+                let s = if &s[..2] == "0x" { &s[2..] } else { &s };
+                let data = hex::decode(s).expect("Failed to decode ActorId");
```

Comment:
We already have the function with similar functionality in relayer. It's called something like `try_from_hex_encoded`. So I guess it would be better to introduce common `utils` crate of the workspace and place the func there